web/components/news-topics-tabs.tsx

```diff
@@ -0,0 +1,79 @@
+import clsx from 'clsx'
+import { ReactNode } from 'react'
+import { Col } from 'web/components/layout/col'
+import { Row } from 'web/components/layout/row'
+import { track } from 'web/lib/service/analytics'
+import { QueryUncontrolledTabs } from 'web/components/layout/tabs'
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+import {
+  UkraineWarData,
+  RedditBlackoutData,
+} from 'web/public/news-topics-data/newsTopicsData'
+
+type Topic = {
+  title: string
+  content: ReactNode
+  onClick?: (event?: any) => void
+}
+
+type NewsTopicsTabsProps = { homeContent: ReactNode }
```

Comment:
Don't define prop types outside the component if they're not being used anywhere else or are small.